dvc/project.py

```diff
@@ -232,6 +232,13 @@ def graph(self):
 
         return G
 
+    def all_directions(self):
```

Comment:
this is fsck-specific function and i don't think it should be placed inside Project class